app/unix/instancemanager.go

```diff
@@ -12,23 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package app
+package unix
 
 import (
 	apiv1 "cloud-android-orchestration/api/v1"
+	"cloud-android-orchestration/app"
 )
 
 // Implements the InstanceManager interface providing access to the first
 // device in the local host orchestrator.
 // This implementation is useful for both development and testing
-type PlaceholderIM struct{}
+type InstanceManager struct{}
 
-func (m *PlaceholderIM) GetHostAddr(_ string, _ string, _ UserInfo) (string, error) {
+func (d *InstanceManager) GetHostAddr(_ string, _ string) (string, error) {
 	return "127.0.0.1", nil
 }
 
-func (d *PlaceholderIM) CreateHost(_ string, _ *apiv1.CreateHostRequest, _ UserInfo) (*apiv1.Operation, error) {
-	return nil, NewInternalError("PlaceholderIM::CreateHost is not implemented", nil)
+func (d *InstanceManager) CreateHost(_ string, _ *apiv1.CreateHostRequest, _ app.UserInfo) (*apiv1.Operation, error) {
+	return nil, app.NewInternalError("unix.InstanceManager::CreateHost is not implemented", nil)
```

Comment:
nit: wondering whether using a C++ nomenclature could be avoided? I think `CreateHost method is not implemented` is fine.

Reply:
I want to include the class name because this will printed without context somewhere else, changed to class#method as it's less language specific

Reply:
sounds good, suggest to use  "%T" format verb then to avoid hard-coding the type.

Reply:
done